docs/platform/2_Custom Cloud Credentials/3_adding-custom-cloud-credentials.md

```diff
@@ -30,6 +30,14 @@ Grid creates clusters designed for large AI workloads. In order to do so, your A
 | EC2 Spot \(instance family you are interested in\) | 1000+ |
 | EC2 On-demand \(instance family you are interested in\) | 1000+ |
 
+Recommended available configuration below for a BYOC cluster, If unavailable please increase your quotas.
```

Comment:
This phrasing is also a bit weird. Maybe:

"Grid will create a number of AWS resources in order to provision your BYOC cluster as seen in the table below. If creating these resources would exceed your quota then the BYOC cluster creation process will fail. In order to address this issue you should either delete existing unused resources or increase your AWS quotas."

Reply:
yeah. That does sound better.